source/git/source.go

```diff
@@ -113,6 +113,26 @@ func (gs *Source) Identifier(scheme, ref string, attrs map[string]string, platfo
 			if v == "true" {
 				id.SkipSubmodules = true
 			}
+		case pb.AttrGitSignatureVerifyPubKey:
+			if id.VerifySignature == nil {
+				id.VerifySignature = &GitSignatureVerifyOptions{}
+			}
+			id.VerifySignature.PubKey = []byte(v)
```

Comment:
I think in the future we probably would also want pubkey via URL, that is then internally loaded in via `llb.HTTP` but I think we can leave that to future iteration.